app/Constants/EmptyTable.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+
+namespace App\Constants;
+
+
+class EmptyTable
+{
+    const WORKLOGS = 'No Worklog(s) Found.';
```

Comment:
this looks good but sometimes this makes code messy and hectic. this is not necessary